docker/optimism/optimism.sh

```diff
@@ -8,8 +8,10 @@ ADDRESS=0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266
 PRIVKEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
 
 CACHE_DIR=$(obtain_cache_dir $0)
-DEPLOYMENT_DIR="${CACHE_DIR}/deployment"
-DEPLOYMENT_CONFIG_FILE="${CACHE_DIR}/optimism-local.json"
+ARTIFACT_DIR="${CACHE_DIR}/deployments/artifacts/local"
+ABI_DIR="${ROOT}/contracts/.build"
```

Comment:
Similarly as for `RPC_FILE`, we could move this to `common.sh`, `deploy_beamer` function.